### ISSUE TITLE
Add stylelint-a11y

### ISSUE THREAD

**Issue by spaceninja** [cloudfour/stylelint-config-cloudfour#208]
Review stylelint-a11y to see if it should be included: https://github.com/YozhikM/stylelint-a11y

**Comment by spaceninja** [comment]
After further review, this plugin doesn't meet our needs. It doesn't support `focus-visible` or our standard practice of disabling transitions globally using the prefers-reduced-motion media query. See #210 for more details, but I'm closing this issue for now.
